bc_obps/reporting/api/report_non_attributable_emissions.py

```diff
@@ -36,18 +39,23 @@ def get_report_non_attributable_by_version_id(
     "/report-version/{version_id}/facilities/{facility_id}/non-attributable",
     response={201: List[ReportNonAttributableOut], custom_codes_4xx: Message},
     tags=EMISSIONS_REPORT_TAGS,
-    description="""Updates given report operation with fields: Operator Legal Name, Operator Trade Name, Operation Name, Operation Type,
+    description="""Updates the given report operation with fields: Operator Legal Name, Operator Trade Name, Operation Name, Operation Type,
```

Comment:
I don't think this description is accurate? (copy paste from somewhere else?)

Reply:
I think added by fixed, fixed.